tests/foreman/ui/test_hw_model.py

```diff
@@ -97,12 +91,35 @@ def test_delete_positive(self, test_data):
 
         """
         with Session(self.browser) as session:
-            bug_id = test_data.pop('bugzilla', None)
-            if bug_id is not None and bz_bug_is_open(bug_id):
-                self.skipTest(
-                    'Bugzilla bug {0} is open for html data.'.format(bug_id)
-                )
+            # See test_hmtl_name() test for skipping 'html' value
+            for name in generate_strings_list(remove_str='html'):
+                with self.subTest(name):
+                    make_hw_model(session, name=name)
+                    self.hardwaremodel.delete(name)
+                    self.assertIsNone(self.hardwaremodel.search(name))
+
+    @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1265150)
+    def test_html_name(self):
+        """@test: To create, update and delete a hw model with html name
```

Comment:
@oshtaier: good question about the need for adding this extra code.  If we need to fix this in generate_strings_list - then we need to account for the entity type and the bugzilla.  If more and more entities start creating problems, generate_strings_list() may become very complex.   helpers need improvement definitely to handle all these different scenarios but it requires more effort.  I am not sure if my datafactory creation will solve this issue, but I will try as part of that effort.